ops/pebble.py

```diff
@@ -810,6 +810,15 @@ def autostart_services(self, timeout: float = 30.0, delay: float = 0.1) -> Chang
         """
         return self._services_action('autostart', [], timeout, delay)
 
+    def replan_services(self, timeout: float = 30.0, delay: float = 0.1) -> ChangeID:
+        """Replan by (re)starting changed and startup-enabled services, then wait for them to be started.
+
+        Raises ChangeError if one or more of the services didn't stop/start. If
+        timeout is 0, submit the action but don't wait; just return the change
+        ID immediately.
+        """
```

Comment:
This applies here, and in a few other places. Docstrings in OF use the Google [docstring style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)

```suggestion
        Raises:
            ChangeError: if one or more of the services didn't stop/start. If
            timeout is 0, submit the action but don't wait; just return the change
            ID immediately.
        """
```